bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -500,16 +522,28 @@ public Bookie(ServerConfiguration conf, StatsLogger statsLogger)
         // configured directories. When disk errors or all the ledger
         // directories are full, would throws exception and fail bookie startup.
         this.ledgerDirsManager.init();
-        // instantiate the journal
-        journal = new Journal(conf, ledgerDirsManager, statsLogger.scope(JOURNAL_SCOPE));
+
+        // instantiate the journals
+        if (journalDirectories.size() == 1) {
+            journals = Lists.newArrayList(new Journal(journalDirectories.get(0), conf, ledgerDirsManager, statsLogger
+                    .scope(JOURNAL_SCOPE)));
+        } else {
+            journals = Lists.newArrayList();
+            for (int idx = 0; idx < journalDirectories.size(); idx++) {
+                journals.add(new Journal(journalDirectories.get(idx), conf, ledgerDirsManager, statsLogger
+                        .scope(JOURNAL_SCOPE + "-" + idx)));
+            }
+        }
```

Comment:
Why are you handling ==1 case differently? Is it just to define the scope? I think even if we have one having scope with idx(0) makes more sense;